app/views/modals/preview-link-modal.html

```diff
@@ -0,0 +1,18 @@
+
+<div class="modal-header">
+  <button type="button" class="close" ng-click="$dismiss();" aria-hidden="true">&times;</button>
+  <h4 class="modal-title">Preview Link Generator</h4>
+</div>
+<div class="modal-body">
+  This link will require loggin in to the CMS to view. It will show all content within this<br>
```

Comment:
Put this text inside a `<p>` tag. Also, shouldn't need `<br>` as the browser figures out where the line break should occur.